### DISCUSSION TITLE
User rights

### DISCUSSION THREAD

**Discussion by jesperordrup** [plausible/analytics#3955] [Anything Else]
Im wondering how users are defined in relation to a server

Example:

A user is invited to a website as viewer
Same user is invited to a website as admin

The user doesnt get any server rights - right?

**Comment by ruslandoga** [comment]
👋 @jesperordrup 

What do you mean by server rights?

I'd need to check how invitations work, but most likely that user's permission would get elevated to admin level for that website. That or the second invitation would fail, because the previous one already exists.

**Comment by jesperordrup** [comment]
> What do you mean by server rights?

Something like, if invited as admin to a website then the user can do more related to creating new sites etc.

**Reply by ruslandoga** [reply]
I think all users, invited or not, can add new sites unless the domains for those sites have already been taken.

**Reply by jesperordrup** [reply]
you're right - thats a problem in itself.  Really want to disable that
